api/specs.proto

```diff
@@ -86,7 +86,7 @@ message ServiceSpec {
 
 	// Service endpoint specifies the user provided configuration
 	// to properly discover and load balance a service.
-	EndpointSpec endpoint = 8;
+	EndpointSpec endpoint = 8 [(gogoproto.nullable) = false];
```

Comment:
Is this needed on a value ?

Reply:
@anshulpundir this is what ensures that it will be an object, not a pointer.